src/pyrad_proc/pyrad/io/io_aux.py

```diff
@@ -417,7 +427,22 @@ def get_file_list(datadescriptor, starttime, endtime, cfg, scan=None):
             dayfilelist = glob.glob(datapath+dayinfo+'*'+datatype+'.nc')
             for filename in dayfilelist:
                 t_filelist.append(filename)
-
+        elif datagroup == 'MXPOL':
+            if scan is None:
+                warn('Unknown scan name')
+                return []
```

Comment:
We do not have a clear naming convention other than trying to respect the suggestions of PEP8:

https://www.python.org/dev/peps/pep-0008/

But we are not particularly disciplined on that :)